lib/tasks/vacuum.rake

```diff
@@ -0,0 +1,34 @@
+namespace :vacuum do
+  desc "Clean up old fiscal year proposals"
+  task old_proposals: :environment do
+    ok_to_act = false
+    if ENV["OK_TO_ACT"] && ENV["OK_TO_ACT"].match(/^[yt]/i)
```

Comment:
I am not sure what the regex value should be in order for this to return `true` -- what about `if ENV["OK_TO_ACT"] -- "true"` ?